blockchain/types/transaction_test.go

```diff
@@ -633,13 +635,21 @@ func TestIntrinsicGas(t *testing.T) {
 		assert.Equal(t, tc.expectGas2, gas)
 		assert.Equal(t, nil, err)
 
-		gas, err = IntrinsicGas(data, nil, true, params.Rules{IsIstanbul: false})
+		gas, err = IntrinsicGas(data, nil, false, params.Rules{IsPrague: true})
 		assert.Equal(t, tc.expectGas3, gas)
 		assert.Equal(t, nil, err)
 
-		gas, err = IntrinsicGas(data, nil, true, params.Rules{IsIstanbul: true})
+		gas, err = IntrinsicGas(data, nil, true, params.Rules{IsIstanbul: false})
```

Comment:
```suggestion
		gas, err = IntrinsicGas(data, nil, true)
```

Reply:
I didn't touch the `expecteGas4`. Additionally, what does this suggestion mean? Fourth parameter is not optional.

Reply:
Oh I misread the code. Nevermind.